src/compiler/tsconfig.json

```diff
@@ -13,10 +13,10 @@
         "corePublic.ts",
         "core.ts",
         "debug.ts",
+        "semver.ts",

```

Comment:
Does this need a comment so it doesn't get alphabetized back?

Reply:
These should never be alphabetized, they are all order dependent.